### DISCUSSION TITLE
Redirection of main page

### DISCUSSION THREAD

**Discussion by NoipName** [thorsten/phpMyFAQ#2787] [Q&A]
Hello everyone,

i need some help with my phpmyfaq

My situation: i configurate my main connection page with some VPN portal (Globalprotect portal) something like this (the picture is just a exemple i found on google):
![image](https://github.com/thorsten/phpMyFAQ/assets/145665195/b8c6dd21-1e9f-45a6-96d3-5a921e31e014)

the problem i have is the main page is not load correctly and dont see anything in the logs can block the main page, same on the firewall:
![image](https://github.com/thorsten/phpMyFAQ/assets/145665195/449fe151-9110-4a62-8902-b4d5af595e7b)

Then i tried to login with MS365 i have error page like this : 
![image](https://github.com/thorsten/phpMyFAQ/assets/145665195/c02fe495-1025-42eb-ab3e-8f21ec52a2b7)

That say SSL PROTOCOL ERROR but i compare it with another working website i have and theses have a same SSL setting

Someone have some idea about this problem ?

**Comment by thorsten** [comment]
Did you configured the SSL domain in phpMyFAQ?

**Reply by NoipName** [reply]
Yep i active all settings of SSL on PhpMyFAQ and SSL on nginx (with ssl certificate,...)
![image](https://github.com/thorsten/phpMyFAQ/assets/145665195/cc0a0f54-2aae-4ece-8525-e66d7df22fb6)
![image](https://github.com/thorsten/phpMyFAQ/assets/145665195/4ef8e97c-26d1-42d4-9238-dfb09b5e790a)

This problem happening just with the VPN Portal, when i use without VPN everything is ok

**Comment by thorsten** [comment]
Could you please try it in incognito mode?

**Comment by thorsten** [comment]
Which SSL certification do you see? The one from the VPN or the phpMyFAQ nginx one?

**Reply by NoipName** [reply]
Incognito mode same issue

The certificat when i'm on the main page is the firewall one, and when i click on connexion azure it didn't have SSL certificate.
And i have this for the Azure connection page https://[phpmyfaq.com](https://phpmyfaq.com/services/azure/)/services/azure/

![image](https://github.com/thorsten/phpMyFAQ/assets/145665195/6a5f3349-9a7c-4cfd-8cfd-07383e2169b1)

**Comment by thorsten** [comment]
But these errors are not from phpMyFAQ
